src/CartesianProduct.jl

```diff
@@ -353,6 +370,53 @@ function ∈(x::AbstractVector{N}, cpa::CartesianProductArray{N, <:LazySet{N}}
     return true
 end
 
+"""
+    constraints_list(cpa::CartesianProductArray{N})::Vector{LinearConstraint{N}} where N<:Real
+
+Return the list of constraints of a (polytopic) Cartesian product.
+
+### Input
+
+- `cpa` -- Cartesian product
+
+### Output
+
+A list of constraints.
+
+"""
+function constraints_list(cpa::CartesianProductArray{N})::Vector{LinearConstraint{N}} where N<:Real
+    # collect low-dimensional constraints lists
+    clist_low = []
```

Comment:
`clist_low = []` is an array of type `Any`. Passing the information known to the compiler will improve efficiency, basically a `clist_low = Vector{LinearConstraint{N}}()`. See [type-stability performance notes](https://docs.julialang.org/en/v1/manual/performance-tips/index.html#Write-%22type-stable%22-functions-1).